bokehjs/src/coffee/models/annotations/label_set.coffee

```diff
@@ -170,7 +170,7 @@ class LabelSet extends TextAnnotation.Model
       angle:        [ p.AngleSpec,    0                 ]
       x_offset:     [ p.NumberSpec,   { value: 0 }      ]
       y_offset:     [ p.NumberSpec,   { value: 0 }      ]
-      source:       [ p.Instance,     () -> new ColumnDataSource.Model()  ]
```

Comment:
was instance of mismatched JS/python defaults (could have been fixed in separate PR but was done here)